scripts/py_matter_idl/matter/idl/test_matter_lint.py

```diff
@@ -0,0 +1,13 @@
+import unittest
+
+
+class TestMatterLinter(unittest.TestCase):
```

Comment:
We have a subset of "check that parsing XML turns into the IDL that we expect" in https://github.com/project-chip/connectedhomeip/blob/master/scripts/py_matter_idl/matter/idl/test_data_model_xml.py

We could also update that one instead.